airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1345,43 +1358,142 @@ def path(
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.forms = Forms(**kwargs)
-
+        self.forms = Forms(**kwargs)        
+    
     def _transform(self, records: Iterable) -> Iterable:
         for record in super()._transform(records):
             keys = record.keys()
 
-            # There's no updatedAt field in the submission however forms fetched by using this field,
-            # so it has to be added to the submissions otherwise it would fail when calling _filter_old_records
+            # There's no updatedAt field in the submission however we need it for saving state
+            # so i added it
             if "updatedAt" not in keys:
                 record["updatedAt"] = record["submittedAt"]
 
+            # Email is a required field on all forms. Might as well add it to the top level
+            # I have to pull the email from their value properties to do that
```

Comment:
It is a personal preference but email is required for every Hubspot form so it made sense to be at the table level. The FiveTran connector had this at this level which was our previous tool (actually, they had contact ID but this is the next best thing); I found it helpful. 

My thought was that since the existing connector only has full refreshes, it wouldn't be a breaking change for most people since the table is rebuilt with every refresh. I'm happy to remove if it will cause issues I didn't anticipate?